src/policies/activityReport.js

```diff
@@ -161,8 +161,20 @@ export default class ActivityReport {
     return approverUserIds.includes(this.user.id);
   }
 
+  // This is a helper function to determine if the report is in a state where it can be edited
   reportHasEditableStatus() {
-    return this.activityReport.submissionStatus === REPORT_STATUSES.DRAFT
-      || this.activityReport.calculatedStatus === REPORT_STATUSES.NEEDS_ACTION;
+    // if the report is in draft, it's editable
+    if (this.activityReport.submissionStatus === REPORT_STATUSES.DRAFT
+      || this.activityReport.calculatedStatus === REPORT_STATUSES.DRAFT) {
```

Comment:
This is essentially damage control - the reports affected by this bug can't be edited if we don't add this little bit of business here. I can't see how it hurts anything - the report stays editable until the hooks have moved the report to "actually submitted" (see calculatedStatus) which is what we check on the frontend as well